packages/web/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "paths": {
       "src/*": ["./src/*"]
     },
-    "allowJs": true
+    "allowJs": false
```

Comment:
This is the only one we have to be careful of, right? Think might be better to leave as true @Tobbe what do you think?

Reply:
This was the last JS code in this project. Everything is TS now, so should be safe to switch over, right?
Looking at other TS-only project (structure, api, auth etc) they don't have `allowJs` specified at all, and the default is `false`. So actually, the correct thing to do here is probably to remove it altogether.

Reply:
Oh well, it's more of a symbolic switch.